ops/mainnet/prod/backend/config.tf

```diff
@@ -72,7 +72,10 @@ locals {
       }
       "1836016741" = {
         providers = ["https://mode-mainnet.blastapi.io/${var.blast_key}", "https://mainnet.mode.network/"]
-      }    
+      }
+      "2020368761" = {
+        providers = ["https://rpc.xlayer.tech", "https://xlayerrpc.okx.com"]
```

Comment:
We should use the unlimited RPC we got from them:
https://rpc.xlayer.tech/unlimited